### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by dAnjou** [isacikgoz/gitbatch#43]
Using version 0.2.0
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1464f6c]

goroutine 40 [running]:
github.com/isacikgoz/gitbatch/pkg/git.(*RepoEntity).loadComponents(0xc00021a100, 0x1, 0xc00021a100, 0x0)
	/Users/ibrahim/go/src/github.com/isacikgoz/gitbatch/pkg/git/repository.go:134 +0xfc
github.com/isacikgoz/gitbatch/pkg/git.InitializeRepo(0xc0000afa20, 0x20, 0xc0000a3d20, 0x0, 0x0)
	/Users/ibrahim/go/src/github.com/isacikgoz/gitbatch/pkg/git/repository.go:104 +0x7d
github.com/isacikgoz/gitbatch/pkg/git.LoadRepositoryEntities.func1(0xc0000a3d20, 0xc0000a3d18, 0xc0001ae040, 0xc0000afa20, 0x20)
	/Users/ibrahim/go/src/github.com/isacikgoz/gitbatch/pkg/git/util-load.go:27 +0x7d
created by github.com/isacikgoz/gitbatch/pkg/git.LoadRepositoryEntities
	/Users/ibrahim/go/src/github.com/isacikgoz/gitbatch/pkg/git/util-load.go:23 +0x158
```

**Comment by isacikgoz** [comment]
gotcha! I think the repository is not on a ref. It's on a commit or something, I'm investigating.

**Comment by isacikgoz** [comment]
fixed it with the pr #44  , please let me know if you have still issues. Thanks for the feedback
